t/venom/test_suites/security_event_random_mac/teardown/05_deconfigure_packetfence.yml

```diff
@@ -0,0 +1,120 @@
+name: Delete configuration in PacketFence
+testcases:
+- name: get_login_token
+  steps:
+  - type: get_login_token
+
+- name: configure_inlinel2_as_other
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/interface/inlinel2'
+    ignore_verify_ssl: true
+    body: >-
+     {
+       "id": "inlinel2",
+       "isClone": false,
+       "isNew": false,
+       "type": "none",
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: delete_user_in_db
+  steps:
+  - type: http
+    method: DELETE
+    url: '{{.pfserver_webadmin_url}}/api/v1/user/iastigmate'
+    ignore_verify_ssl: true
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: deconfigure_snat_interface_for_passthroughs
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/base/network'
+    ignore_verify_ssl: true
+    body: >-
+      {
+        "interfaceSNAT":""
+      }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+      - result.bodyjson.message ShouldEqual "Settings updated"
+
+- name: deconfigure_snat_interface_for_inline
+  steps:
+  - type: http
+    method: PATCH
+    url: '{{.pfserver_webadmin_url}}/api/v1/config/base/inline'
+    ignore_verify_ssl: true
+    body: >-
+     {
+       "id": "inline",
+       "interfaceSNAT": ""
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: desable_security_event
+  steps:
+  - type: pf_api_action
+    method: PATCH
+    url: 'config/security_event/{{.security_event_random_mac.event.id}}'
+    body: '{"id":"{{.security_event_random_mac.event.id}}","enabled":"N","quiet":true}'
+
+
+- name: restart_iptables
```

Comment:
All services below will need to be restarted through API to be consistent with changes made on https://github.com/inverse-inc/packetfence/pull/7367